tests/test_proxy.py

```diff
@@ -6,6 +6,55 @@
 from django_docker_engine.historian import FileHistorian
 
 
+class CSRFTests(unittest.TestCase):
+
+    def setUp(self):
+        self.fake_get_kwargs = {
+            'request': RequestFactory().get('/fake-url'),
+            'container_name': 'fake-container',
+            'url': 'fake-url'
+        }
+        self.fake_post_kwargs = {
+            'request': RequestFactory().post('/fake-url'),
+            'container_name': 'fake-container',
+            'url': 'fake-url'
+        }
+    # TODO?
+    #     timestamp = re.sub(r'\W', '-', datetime.now().isoformat())
+    #     data_dir = '/tmp/django-docker-tests-' + timestamp
+    #     self.root_label = 'test-root'
+    #     self.manager = DockerEngineManager(data_dir, self.root_label)
+    #     self.container_name = timestamp
+    #     self.manager.run('nginx:1.10.3-alpine',
+    #                      ports={'80/tcp': None},
+    #                      detach=True,
+    #                      labels={'test-root.port': '80'},
+    #                      cmd=None)
+
+    def test_csrf_exempt_true(self):
```

Comment:
Try to have stronger assertion about csrf or lack of csrf presence for the following tests